### PR TITLE
l10n: Fix text string

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]

**Review by rakekniven — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by Valdnet** [comment]
Please merge.

**Comment by Valdnet** [comment]
@nickvergessen Please merge. It would be good if the amendment was also added to the NC 25 version.

File: apps/files/js/app.js [hunk 1]
```diff
@@ -114,13 +114,13 @@
 						OCA.Files.FileList.MultiSelectMenuActions.ToggleSelectionModeAction,
 						{
 							name: 'delete',
-							displayName: t('files', 'Delete'),
+							displayName:  t('files', 'Delete'),
```
Comment (nickvergessen) [inline apps/files/js/app.js:117]:
```suggestion
							displayName: t('files', 'Delete'),
```

File: apps/files/js/app.js [hunk 2]
```diff
@@ -114,13 +114,13 @@
...
 							iconClass: 'icon-delete',
 							order: 99,
 						},
 						{
 							name: 'tags',
-							displayName:  'Tags',
+							displayName:  t('files', 'Tags'),
```
Comment (nickvergessen) [inline apps/files/js/app.js:123]:
```suggestion
							displayName: t('files', 'Tags'),
```

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**